### ISSUE TITLE
Feature: Add WCAG or best practice criteria to the report

### ISSUE THREAD

**Issue by maggiewachs** [mgifford/open-scans#123]
While it's helpful to see which rules pass/fail, I need to know the associated WCAG or best practice criteria in order to:
* Determine remediation priority for failures. I might prioritize an issue that fails Level A criteria over one that fails a best practice.
* Use the results in an ACR, which only tracks WCAG / Section 508 criteria.

This info would be useful as a reference table, and/or alongside each rule failure Pattern (rule: `image-alt`, criteria: WCAG SC 1.1.1). It may also be helpful to include Section 508 FPC.
